app/src/main/java/com/gh4a/activities/RepositoryActivity.java

```diff
@@ -182,24 +131,42 @@ protected void onResultReady(Boolean result) {
     private ContentListContainerFragment mContentListFragment;
     private CommitListFragment mCommitListFragment;
     private RepositoryEventListFragment mActivityFragment;
+    private Gh4Application mApp;
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        mApp = (Gh4Application) getApplicationContext();
 
         mActionBar = getSupportActionBar();
         mActionBar.setTitle(mRepoOwner + "/" + mRepoName);
         mActionBar.setDisplayHomeAsUpEnabled(true);
 
         setContentShown(false);
 
-        getSupportLoaderManager().initLoader(LOADER_REPO, null, mRepoCallback);
+        this.loadRepository().subscribe(o -> {}, e -> {});
```

Comment:
Code style: We do not prefix method with `this`. Applies to other places in code too.